native/src/modules/layout/components/TransparentHeader.js

```diff
@@ -85,18 +84,20 @@ class TransparentHeader extends React.PureComponent<PropsType> {
   }
 
   render () {
-    const { theme, float, navigation, t } = this.props
-    const shareUrl = navigation.state.params.url
+    const { theme, float, scene, t } = this.props
+    const shareUrl = scene.route.params?.shareUrl || ''
```

Comment:
I think it makes more sense to pass null if the share path is undefined instead of an empty string.

Reply:
Will fix in #105